civis/ml/_model.py

```diff
@@ -639,6 +653,23 @@ def __init__(self, model, dependent_variable,
         self._client = client
         self.train_result_ = None
 
+        if '_NEWEST_CIVISML_VERSION' not in globals():
+            global _NEWEST_CIVISML_VERSION
+            try:
+                client.templates.get_scripts(id=self.train_template_id)
+                client.templates.get_scripts(id=self.predict_template_id)
```

Comment:
It seems like this block of new code (L656 through L671) would work well as a new method to `ModelPipeline`. There should also be a comment to explain why you're calling `templates.get_scripts` twice and ignoring the outputs.

You'll need to call this method ("_set_template_version"?) in `__setstate__` below. Users can unpickle models without going through the constructor.